modules/geopackage/src/lib/parse-geopackage.js

```diff
@@ -115,11 +147,12 @@ function getVectorTable(db, tableName, projections, {reproject, _targetCrs}) {
     geojsonFeatures.push(geojsonFeature);
   }
 
+  const schema = getArrowSchema(db, tableName);
   if (projection) {
-    return transformGeoJsonCoords(geojsonFeatures, projection.project);
+    return {geojsonFeatures: transformGeoJsonCoords(geojsonFeatures, projection.project), schema};
   }
 
-  return geojsonFeatures;
+  return {geojsonFeatures, schema};
```

Comment:
Note this is a breaking change because the final output from `parseGeoPackage` will change from
```js
{
	'table1': [features]
}
```
to
```js
{
	'table1': {
		'schema': schema,
		'geojsonFeatures': [features]
	},
	'table2' ...
}
```

It might be better if we standardized a geospatial output format for loaders to include both features and metadata. Might be good to discuss before the 3.0.0 release @ibgreen

Reply:
Good point, let me look into that.